azure/template.json

```diff
@@ -177,6 +177,10 @@
                                 {
                                     "name": "EnvironmentName",
                                     "value": "[parameters('environmentName')]"
+                                },
+								{
```

Comment:
Could the indentation be fixed for the ResourceEnvironmentName app settings please

Reply:
Sorted.